src/core/software/pupil-player.md

```diff
@@ -509,6 +514,20 @@ it is recommended to use primitive types (strings, integers, floats) as value ty
 custom fields.
 :::
 
+Annotation hotkey definitions are stored in the recording directory: 
+`<recording dir>/offline_data/annotation_definitions.json` with the format:
+```json
+{
+    "version": 1,
+    "definitions": {
+        "<label>": "<hotkey>"
+    }
+}
+```
+When a new recording is loaded, Pupil Player will attempt to load the annotation definitions from the recording-specific 
+file. If it is not found or invalid, the last known set of annotation definitions will be loaded from Pupil Player's 
+session settings. This ensures you never lose your annotation hotkey definitions.
```

Comment:
```suggestion
session settings.
```
Given that session settings reset on app version upgrades, I would not make this statement.

Reply:
Agree